projects/ng-components/lib/toggle-selector/toggle-selector.component.html

```diff
@@ -0,0 +1,28 @@
+<ng-container *ngFor="let item of displayList; let idx = index; trackBy: trackByFn">
+    <div
+        class="ac toggle-selector"
+        [class.error]="error"
+        [class.checked]="item.selected"
+        [class.disabled]="disabled || item.optionItem.disabled"
+        [class.focus]="focusedItem === idx"
+        (click)="handleItemOnClick($event, idx)"
+    >
+        <input
+            [type]="inputType"
```

Comment:
I don't think there is a need for a getter here. It's quite straightforward
```suggestion
            [type]="multi ? 'checkbox' : 'radio'"
```

Reply:
(Russian accent) but but but i created this type: 
```typescript
export type InputType = "checkbox" | "radio";
```